protoc-gen-openapiv2/defs.bzl

```diff
@@ -174,25 +214,81 @@ protoc_gen_openapiv2 = rule(
             mandatory = True,
             providers = [ProtoInfo],
         ),
-        "grpc_api_configuration": attr.label(
-            allow_single_file = True,
+        "single_output": attr.bool(
+            default = False,
             mandatory = False,
+            doc = "if set, rule will generate single OpenAPI file",
         ),
-        "single_output": attr.bool(
+        "allow_delete_body": attr.bool(
             default = False,
             mandatory = False,
+            doc = "unless set, HTTP DELETE methods may not have a body",
+        ),
+        "grpc_api_configuration": attr.label(
+            allow_single_file = True,
+            mandatory = False,
+            doc = "label gRPC API Configuration in YAML format",
         ),
         "json_names_for_fields": attr.bool(
             default = True,
             mandatory = False,
+            doc = "if set, the original proto name will be used for" +
+                  " generating OpenAPI definitions",
```

Comment:
I'm not sure this is right? If it's set (the default), it should use the json names for the openapi generated files.

Reply:
Mea culpa. fixed.